### PR TITLE
Add support for `calc-size()`

### PR DESCRIPTION
**Pull request by nex3**
See sass/sass#3989
See sass/sass-spec#2039

### CODE REVIEW

File: lib/src/value/calculation.dart [hunk 1]
```diff
@@ -604,6 +604,21 @@ final class SassCalculation extends Value {
...
 
+  /// Creates an `calc-size()` calculation with the given [basis] and [value].
+  ///
+  /// The [basis] and [value] must be either a [SassNumber], a
+  /// [SassCalculation], an unquoted [SassString], or a [CalculationOperation].
+  ///
+  /// This automatically simplifies the calculation. It throws an exception if
+  /// it can determine that the calculation will definitely produce invalid CSS.
+  static SassCalculation calcSize(Object basis, Object? value) {
+    var args = [basis, if (value != null) value];
```
Comment (pamelalozano16) [inline lib/src/value/calculation.dart:615]: Should you check for types here? That they are either a `SassNumber`, a `SassCalculation`,  `SassString`, or a `CalculationOperation`?

File: lib/src/value/calculation.dart [hunk 2]
```diff
@@ -604,6 +604,21 @@ final class SassCalculation extends Value {
     }
   }
 
+  /// Creates an `calc-size()` calculation with the given [basis] and [value].
+  ///
+  /// The [basis] and [value] must be either a [SassNumber], a
```
Comment (pamelalozano16) [inline lib/src/value/calculation.dart:609]: Value is optional right? So it can also be null

**Review by pamelalozano16 — APPROVED** [review]

File: lib/src/value/calculation.dart (hunk 1 above)
Comment (nex3) [reply lib/src/value/calculation.dart:615]: That's checked by `_simplify()`.

File: lib/src/value/calculation.dart (hunk 2 above)
Comment (nex3) [reply lib/src/value/calculation.dart:609]: Yes, but that's implicit from the type.

**Pull request merged**